app/src/main/java/com/example/wellfed/ActivityBase.java

```diff
@@ -16,4 +16,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return super.onOptionsItemSelected(item);
     }
+
+    public abstract void deleteIngredient();
+
+    public abstract void delete();
```

Comment:
revert this, and all the uses in the subclasses

Reply:
I've made those changes, can you check it now?

Reply:
LGTM